conda-forge.yml

```
@@ -4,3 +4,9 @@ github:
 conda_forge_output_validation: true
 conda_build:
   pkg_format: '2'
+conda_build_tool: rattler-build
+conda_install_tool: micromamba
```

Maybe we can try `conda`

```suggestion
conda_install_tool: conda
```

Also please make sure to re-render after changing this

---

Good idea but same error with conda

```
 │ ╭─ Fetching source code
 │ │ Fetching source from git repo: https://github.com/lightvector/KataGo.git
 │ │ Fetching repository from https://github.com/lightvector/KataGo.git at refs/tags/v1.15.3 into D:\bld\src_cache\KataGo.git
 │ │ × error Command failed: `git rev-parse "refs/tags/v1.15.3"`
```

Logs: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1158244&view=logs&j=f4756218-2105-5c4f-5cb9-a0ac2817b29f&t=618de3fb-810d-5a79-16ac-89d47be0c182